storage_service/administration/forms.py

```diff
@@ -222,8 +222,17 @@ class UserChangeForm(auth.forms.UserChangeForm):
     """ Modifys an existing user.  Inherits from django's UserChangeForm. """
 
     def __init__(self, *args, **kwargs):
+        self.current_user = kwargs.pop("current_user", None).username
         super(UserChangeForm, self).__init__(*args, **kwargs)
+        currently_editing = getattr(kwargs["instance"], "username")
+        superusers = auth.get_user_model().objects.filter(is_superuser=True)
         self.fields["is_superuser"].label = _("Administrator?")
+        if len(superusers) >= 1:
+            if (
+                not getattr(kwargs["instance"], "is_superuser")
+                and self.current_user == currently_editing
```

Comment:
I don't think we need this comparison (see note on review) but also a common Django thing here: you can just compare user objects with `==`, so `kwargs.pop("current_user", None) == kwargs["instance"]` (as long as they're not both None). It's mostly a matter of preference, but that method will compare the primary keys.